tests/entities/test_collection.py

```diff
@@ -0,0 +1,145 @@
+from bibx._entities.article import Article
+from bibx._entities.collection import Collection
+
+
+class TestCollection:
```

Comment:
The `class` can also be omited, just the function called `test_something` is enough.

Reply:
:+1: